src/main/kotlin/com/devfloor/untitled/article/application/ArticleResponse.kt

```diff
@@ -0,0 +1,20 @@
+package com.devfloor.untitled.article.application
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import java.time.LocalDateTime
+
+data class ArticleResponse(
+    val options: List<String>,
+    val title: String? = null,
+    val anonymous: Boolean,
+    val content: String,
+    val author: Any,
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", shape = JsonFormat.Shape.STRING)
+    val createdDate: LocalDateTime,
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", shape = JsonFormat.Shape.STRING)
+    val modifiedDate: LocalDateTime,
+    val hashtags: List<String>,
+    val favorites: Long,
+    val wonders: Long,
+    val clips: Long
+)
```

Comment:
저거 어노테이션 들어가면서 약간 프로퍼티 별 구분이 필요할 것 같아서 사이에 공백을 넣어주는건 어떨까?

```suggestion
data class ArticleResponse(
    val options: List<String>,
    
    val title: String? = null,
    
    val anonymous: Boolean,
    
    val content: String,
    
    val author: Any,
    
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", shape = JsonFormat.Shape.STRING)
    val createdDate: LocalDateTime,
    
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", shape = JsonFormat.Shape.STRING)
    val modifiedDate: LocalDateTime,
    
    val hashtags: List<String>,
    
    val favorites: Long,
    
    val wonders: Long,
    
    val clips: Long
)
```

Reply:
반영완료 !